pkg/etcd/etcd.go

```diff
@@ -89,11 +99,11 @@ func (e *ETCD) IsInitialized(ctx context.Context, config *config.Control) (bool,
 	}
 }
 
-func (e *ETCD) Reset(ctx context.Context) error {
+func (e *ETCD) Reset(ctx context.Context, clientaccess *clientaccess.Info) error {
 	go func() {
 		for {
```

Comment:
I believe it's preferred to create a ticker (defer stop it) and to range over that resulting channel in the for loop itself rather than using the time.Sleep method. For here and the method below.